registry/registry.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/docker/distribution/registry/client/transport"
 	"github.com/docker/go-connections/tlsconfig"
 	"github.com/sirupsen/logrus"
+
+	"github.com/docker/docker/pkg/homedir"
```

Comment:
nit: these imports should be in the same group as the previous one (not a blocker)

Reply:
okay, thanks.

Reply:
okay, thanks